DynatraceAppMon/Context.cs

```diff
@@ -330,6 +430,38 @@ public ArrayList getMethod(Lookup lookup, List<Project> projects)
             return codeFunctions;
         }
 
+        private void FindClasses(Lookup lookup, ProjectItems projectItems, ref List<String> hits)
+        {
+            foreach (ProjectItem item in projectItems)
+            {
+                try
+                {
+                    if (item.ProjectItems != null) FindClasses(lookup, item.ProjectItems, ref hits);
+                    else if (!HasClassFiles(item)) continue;
```

Comment:
If it is easy to avoid continue, break etc. it is good to do so. I mean when control flow is not jumping it is easier to read